spec/release/notes/link_spec.rb

```diff
@@ -3,4 +3,24 @@
 require "spec_helper"
 
 describe Release::Notes::Link do
+  after { restore_config }
+
+  class TestClass
+    include Release::Notes::Link
+    attr_reader :config
+
+    def initialize(config)
+      @config = config
+    end
+  end
+
+  describe "#log" do
```

Comment:
Not testing anything? Guessing this is coming in the other PR though...

Reply:
https://github.com/dvmonroe/release-notes/pull/12